sample-storage/mock-connections.yaml

```diff
@@ -0,0 +1,115 @@
+# Credentials to the test databases.
```

Comment:
This file should be in a different folder and it must be named `connections.yaml`, otherwise `getQueries` won't load the file.

See my comment for `test/backend/datastores.mock.spec.js`.